### DISCUSSION TITLE
Plex not populating Collections

### DISCUSSION THREAD

**Discussion by lawllipops21** [meisnate12/Plex-Meta-Manager#680] [Q&A]
Hi Everyone,

I am able to get the script to run, but when i go into Plex i dont see any of the collections being added.

I did see in the logs 
Connecting to Movies Library...                                                                                                                                                                        
[2022-02-01 13:20:30,479] [util.py:162]               [DEBUG]    | Config Warning: missing_path attribute not found using None as default   

ive uploaded my logs as well.

NOTE 
i tried changing the advanced setting in my plex library to use The Movie Database, and had me upgrade my libraries, wasnt sure what I needed to set it as.

[meta.log.1.txt](https://github.com/meisnate12/Plex-Meta-Manager/files/7982188/meta.log.1.txt)

**Comment by kevbentz** [comment]
Did you edit the config.yml file?

**Reply by meisnate12** [reply]
this was discussed on discord im pretty sure it was fixed by reloading the plex container

**Reply by lawllipops21** [reply]
yeah! i was able to get help on the Discord Server. Thanks!

we werent able to figure out what the issue was but it was failing to get imported to plex.  I basically started over and reinstalled plex and PMM and it worked.

also i dont know how to mark this issue as resolved or answered.
